build.sh

```diff
@@ -1,8 +1,44 @@
 #!/bin/bash
 
-mkdir -p ./build
 
-# Build the conditions
+TEMP=$(getopt -o o: --long configs:,conditions:,output: \
```

Comment:
what is this? 😬

Reply:
`getopt` was apparently necessary when I wanted to add arguments with long names to the script. The arguments themselves where necessary to change set of data for testing.

I can either add some comment to explain this, or remove CLI options and use something else, like env variables.

Reply:
just add a comment :) TY!

Reply:
Added :)